pkg/aws/metadata/metadata.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Authors of Cilium
+// Copyright 2021 Authors of Cilium
```

Comment:
Typically we keep the year ranges of the copyright:

```suggestion
// Copyright 2019-2021 Authors of Cilium
```